docs/sources/release-notes/cadence.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Release cadence
+description: How our release process works
+weight: 1
+---
+
+# Release cadence
+
+## Stable Releases
+
+Loki releases (this includes [Promtail](/clients/promtail), [Loki Canary](/operations/loki-canary/), etc) use the following
+naming scheme: `MAJOR`.`MINOR`.`PATCH`
+
+- `MAJOR` (roughly once a year): these releases include large new features & possible backwards-compatibility breaks
+- `MINOR` (roughly once a quarter): these releases include new features which do not break backwards-compatibility
```

Comment:
Depending on how we defined our stability contract (what does it include) this isn't entirely true.

We do make breaking changes in minor releases from time to time, mostly around configs.

I think the note below sums it up best, we will over communicate and do our best to make upgrades as painless as possible, always. But we may make some changes considered breaking in minor releases.